gptqmodel/looper/module_looper.py

```diff
@@ -105,7 +105,10 @@ def store_input_hook(_, args, kwargs):
         self.gptq_model.pre_quantize_generate_hook_start()
         for example in calibration_data:
             for k, v in example.items():
-                data_device = self.gptq_model.quantize_config.device if k == "pixel_values" else cur_layer_device
+                if self.gptq_model.__class__.__name__ == 'Qwen2_5_OmniGPTQ':
```

Comment:
@tiger-of-shawn Can we do an explicit instance of or inhereitance check instead of class name string check here? I want to avoid regressions if we decide to rename class file names in the future.